ej2-angular/progress-button/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with Angular Progress button component | Syncfusion
-description:  Checkout and learn about Getting started with Angular Progress button component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with Angular ProgressBar Component | Syncfusion
```

Comment:
Change like this - Getting Started with Angular Progress Button Component | Syncfusion
check and change in all places - description and H1

Reply:
resolved

Reply:
Still not resolved yet